src/app/shared/error.service.ts

```diff
@@ -25,9 +25,7 @@ export class ErrorService {
         let errorObj = null;
         if (error) {
             errorObj = {
-                message: 'The webservice encountered an error trying to create/modify.',
```

Comment:
Will this change apply for all error codes or just when the web service is down?
I'd like bad requests, internal server errors, etc. to be reported as normal but when the server is completely down (as in the linked issue), this applies

Reply:
I'm almost positive that this will change all error messages.  Also, if the webservice is known to be down and not a legit bad request, internal server error, it should just redirect to maintenance page.

Reply:
Yes looking more into it this is the case. I'll instead show a different error message if the HTTP response is 0 (no response).